RELEASE_PROCESS.md

```diff
@@ -27,43 +27,123 @@ In order to release, you will need the following accounts/permissions:
+- Merge to master.  
+Additional context:  After a commit is merged to master, a build is automatically kicked-off on the Chef / [chef/chef-server:master] omnibus/release pipeline by expeditor after it bumps the version number.  After the build and tests in buildkite pass, expeditor should put the build artifact in Artifactory's current channel.
+- Make sure the omnibus build to be promoted is present in Artifactory's current channel.  
+One approach is to enter the following into a bash shell, where _version_ is the version number of the new release:
+```
+$ mixlib-install download chef-server -c current -a x86_64 -p ubuntu -l 16.04 -v <version>
+Starting download https://packages.chef.io/files/current/chef-server/14.1.0/ubuntu/16.04/chef-server-core_14.1.0-1_amd64.deb
+```
+- Make sure the Habitat builds for master are passing.  
+  Chef / [chef/chef-server:master] habitat/build / master  
+  https://buildkite.com/chef/chef-chef-server-master-habitat-build
+
+Monitor the chef-server-notify slack channel for current progress. [QUESTION: where does this sentence go, i.e. at what step do we monitor progress?  During the build?]
 
 ### Testing the Release
 
-> This process is currently manual. There is working being done to fold this into a testing pipeline.
-
-Every merge to chef-server master is built and this clean build is tested
-with all pedant tests. We only run smoke tests for the FIPS mode. Upgrade
-and addon testing must be done in advance of the release.
-
-- [ ] Test as per the matrix from: https://docs.google.com/spreadsheets/d/1_gwxdrMnUiV8t2noi8zs1HyGivtIGMEx9KbYriIw7BY/edit#gid=536218507
-
-- [ ] If this release is being made to address a specific
-  high-urgency, high-severity customer issue or security issue, please
-  *specifically* test that the issue in question is fixed.
-
-If one of these tests has failed, you cannot ship a release until it's fixed.
-Note, no changes other than CHANGELOG/RELEASE_NOTES changes should
-land on master between testing and releasing since we typically tag
-HEAD of master. If something large does land on master, the release
-tag you create should point specifically at the build that you tested.
-The git SHA of the build you are testing can be found in
-`/opt/opscode/version-manifest.json`.
-
-### Preparing for the release
+Every merge to chef-server master must be built, and this build must be tested with the full Umbrella automated integration test pipeline at https://buildkite.com/chef/chef-umbrella-master-chef-server-full.
+The integration test run for the tag being shipped must be successful.
+
+Any Chef Infra Server release after 12.17.35 [QUESTION: inclusive of?] should be able to upgrade directly to the new release.  Older releases were already tested during previous release processes, but make sure to test the stable release being replaced by this new release, and...  
+[...anything else?  last release we tested the following:
+14.1.0  -> 14.2.2
+14.0.65 -> 14.2.2
```

Comment:
the nightlies test upgrades to the latest current artifact from 12.17.15 and 13.2.0